src/misc/misc.h

```diff
@@ -52,12 +54,24 @@ extern void warn_nonnull_ptr(void*);
 extern int parse_cfl(_Complex float res[1], const char* str);
 #ifndef __cplusplus
 extern noreturn void error(const char* str, ...);
+#elif defined __clang__
+extern _Noreturn void error(const char* str, ...);
```

Comment:
What for? The standard is noreturn (which expands to _Noreturn=

Reply:
I had some compilation problems which have since disappeared...